test/lib/processor_spec.js

```diff
@@ -206,7 +209,51 @@ describe('Processor', function() {
             done();
           });
       });
-  });
+    });
+
+    describe("when preprocessor fails", function() {
+      var error = new Error("Test");
+
+      beforeEach(function() {
+        sinon.stub(preprocessor.prototype, 'process', function(file_or_stream, outfile, callback) {
+          callback(error);
+        });
+      });
+
+      afterEach(function() {
+        preprocessor.prototype.process.restore();
+      });
+
+      it('should return error', function(done) {
+        scanner(__dirname + "/../test_files/readable.jpg")
+        .parse(function(err, results) {
+          assert.equal(error, err);
+          done();
+        });
+      });
+    });
+
+    describe("when tesseract fails", function() {
```

Comment:
This should probably be `when tesseract returns error`, but then the test seems meaningless: `process, when tesseract returns error, it returns error`.